app/scripts/modules/core/src/pipeline/executions/execution/ExecutionMarker.tsx

```diff
@@ -80,13 +111,40 @@ export class ExecutionMarker extends React.Component<IExecutionMarkerProps, IExe
         <span className="horizontal center middle">
           <MarkerIcon stage={stage} />
           <span className="duration">{this.state.duration}</span>
+          {SETTINGS.feature.executionMarkerInformationPopover &&
+            stage.status.toLowerCase() === 'terminal' && stage.type === 'pipeline' && (
+              <i
+                className="fa fa-info-circle"
+                style={{
+                  color: '#fff',
+                  marginLeft: '2px',
+                  fontSize: '.8em',
+                  position: 'relative',
+                  top: '-1px',
+                }}
+                onClick={this.handleStageInformationClick}
+                key={`${execution.id}_${stage.id}`}
+                data-id={`${execution.id}_${stage.id}`}
```

Comment:
since i'm passing in the the actual event target, i could look at passing the execution and stage to the information component and using that. i would have to look again i'm not even sure if i'm using data-* for anything anymore.

Reply:
@maggieneterval data-* has been removed using the target i was passing in :)

Reply:
Looks like you've removed the data attribute 👍